components/sup/src/manager/mod.rs

```diff
@@ -664,7 +664,16 @@ impl Manager {
         });
         core.handle().spawn(ctl_handler);
         if let Some(svc_load) = svc {
-            Self::service_load(&self.state, &mut CtlRequest::default(), svc_load).unwrap();
+            match Self::service_load(&self.state, &mut CtlRequest::default(), svc_load) {
+                Err(err) => {
+                    if err.get_code() == ErrCode::Conflict {
```

Comment:
Doesn't this swallow the error when you try to load something that is already loaded, without passing `--force`?

Reply:
Ah, I can see a slight edge case here now that you point it out.

This only runs if you pass a package identifier to `hab sup run`. However, if you had previously ever run the Supervisor and loaded a service with the same name, we'd just use what you had specified for that service to run and ignore any additional flags that you passed to `hab sup run`.

I don't think people want to need to pass `--force` here and `hab sup run` is more of a do what I mean thing. If that's the case they'd probably be wondering why the other flags they passed didn't get picked up. I'll think a bit more about this